code/game/gamemodes/miniantags/slaughter/slaughterevent.dm

```diff
@@ -2,9 +2,9 @@
 /datum/round_event_control/slaughter
 	name = "Spawn Slaughter Demon"
 	typepath = /datum/round_event/slaughter
-	weight = 0 //Admin only
+	weight = 1 //Very rare
 	max_occurrences = 1
-	earliest_start = 30 //Spawn later with more blood, hopefully
+	earliest_start = 0
```

Comment:
I recommend keeping this at 30 minutes, it would be rather boring to wind up as a demon with a still clean station. Also, this may interfere with the progress of certain antagonists.